scalafix/README.md

```diff
@@ -14,12 +14,22 @@ run
 sbt scalafix github:typelevel/cats/v1.0.0
 ```
 
+to run all rules that apply to version `1.0.0`.
+
+If you only want to run the rules that apply to version `1.0.0-MF` run
+
+```sh
+sbt scalafix github:typelevel/cats/v1.0.0?sha=c131fe4
```

Comment:
The 1.0.0 update on master has not much practical use now since it's not released, how about we reorder the paragraphs to  this? 
```markdown
to run the rules that apply to version 1.0.0-MF run

sbt scalafix github:typelevel/cats/v1.0.0?sha=c131fe4

to run all rules that apply to current 1.0.0-SNAPSHOT

sbt scalafix github:typelevel/cats/v1.0.0

```
We can update it again once we release 1.0.0-RC1

Reply:
Sounds good